pubspec.lock

```diff
@@ -1,6 +1,27 @@
 # Generated by pub
```

Comment:
I would not put this file in source control (you could keep it only for the main application but I would say not for libraries)

Reply:
it's for the main app I think, I removed from the library.